### PR TITLE
Escape single quote in message format

### PR DESCRIPTION
**Pull request by IvanBorislavovDimitrov**

### CODE REVIEW

File: multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java [hunk 1]
```diff
@@ -83,11 +83,11 @@ static Stream<Arguments> testGetAllDeployedMtas() {
                          // (2) Applications without module in metadata:
                          Arguments.of("metadata/apps-02.json", null,
                                       new Expectation(Expectation.Type.EXCEPTION,
-                                                      "MTA metadata for entity \"mta-application-1\" is incomplete. This indicates that MTA reserved variables in the entitys metadata were modified manually. Either revert the changes or delete the entity.")),
+                                                      "MTA metadata for entity \"mta-application-1\" is incomplete. This indicates that MTA reserved variables in the entity's metadata were modified manually. Either revert the changes or delete the entity.")),
```
Comment (radito3) [inline multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java:86]: Shouldn't this be escaped as well?

File: multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java [hunk 2]
```diff
@@ -83,11 +83,11 @@ static Stream<Arguments> testGetAllDeployedMtas() {
...
                          // (3) Services without resource in metadata:
                          Arguments.of("metadata/apps-03.json", "metadata/services-03.json",
                                       new Expectation(Expectation.Type.EXCEPTION,
-                                                      "MTA metadata for entity \"mta-service-1\" is incomplete. This indicates that MTA reserved variables in the entitys metadata were modified manually. Either revert the changes or delete the entity.")),
+                                                      "MTA metadata for entity \"mta-service-1\" is incomplete. This indicates that MTA reserved variables in the entity's metadata were modified manually. Either revert the changes or delete the entity.")),
```
Comment (radito3) [inline multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java:90]: Same here

File: multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java (hunk 1 above)
Comment (IvanBorislavovDimitrov) [reply multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java:86]: No, this is a test message

File: multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java (hunk 2 above)
Comment (IvanBorislavovDimitrov) [reply multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/cf/detect/DeployedMtaDetectorTest.java:90]: No, this is a test message

**Review by radito3 — APPROVED** [review]

**Pull request merged**